server/controllers/inventory/index.js

```diff
@@ -88,6 +93,98 @@ function inventoryLog(req, res, next) {
   }).catch(next);
 }
 
+// get inventory log as excel
+// GET /inventory/download/log/:uuid?rendere=xlsx?lang=fr
+
+async function logDownLoad(req, res, next) {
+  try {
+    const { lang } = req.query;
+    const rows = await core.inventoryLog(req.params.uuid);
+    // inventory columns
+    const cols = {
+      code : 'FORM.LABELS.CODE',
+      consumable : 'FORM.LABELS.CONSUMABLE',
```

Comment:
This `const` isn't dependent on this function at all.  Could you move it out of the function and make it a global variable?